db/migrate/20210618132206_add_notification_settings.rb

```diff
@@ -22,106 +22,13 @@ def up
               name: 'index_notification_settings_unique_project'
     end
 
-    insert_project_specific_channel
-    insert_default_mail_channel
-    insert_default_in_app_channel
-
     remove_column :members, :mail_notification
```

Comment:
Not really related to this PR but the `mail_notification` column of the `users` table is never deleted when it should. The down migration recreates it at least.